client/client.go

```diff
@@ -30,14 +30,27 @@ type Instance struct {
 	DatabaseEngine     EngineType `json:"databaseEngine"`
 }
 
-// OfferType is the charging type of the price
+// ChargeType is the charge type of the price
+type ChargeType string
+
+const (
+	// ChargeTypeOnDemand is the on demand type of the price
```

Comment:
nit: complete with period in the end if this is a full sentence. And fix other places.